src/components/WelcomeBox.css

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
+ :root[platform="mac"] .welcomebox .shortcutKey,
```

Comment:
This rule should be placed after the styles for `.welcomebox .shortcutKey`, to keep a logical order.

Reply:
Do you think now it's the correct place? :)

Reply:
Looks like it. :)